__tests__/integration/classes.test.js

```diff
@@ -1,10 +1,13 @@
 import request from 'supertest';
+import mongoose from 'mongoose';
 import app from '../../src/app';
 import Class from '../../src/app/schemas/Class';
 
 describe('Classes', () => {
-	afterAll(async () => {
+	afterAll(async (done) => {
 		await Class.deleteMany();
+		await mongoose.connection.close();
+		done();
```

Comment:
Is it necessary to call `done()` here?
I was reading [Jest documentation](https://github.com/centralufabc/central-backend/pull/9/files#diff-f1a27f9e8e2aeb807b5cfa1bb369470fR9) today and it seems like using just the `async / await` combo is enough.

Reply:
you're right, i removed it.